keras_rcnn/layers/object_detection/object_proposal.py

```diff
@@ -0,0 +1,59 @@
+import keras.backend
```

Comment:
The module should be `_object_proposal.py` rather than `object_proposal.py`.

This will fix the current Travis failure.